libs/perun/components/src/lib/members-dynamic-list/members-dynamic-list.component.ts

```diff
@@ -165,34 +186,45 @@ export class MembersDynamicListComponent implements AfterViewInit, OnInit, OnCha
 
   changeStatus(event: Event, member: RichMember, groupId?: number): void {
     event.stopPropagation();
-    const config = getDefaultDialogConfig();
-    config.width = '500px';
-    config.data = { member: member, voId: this.voId, groupId: groupId };
-    const oldStatus = groupId ? member.groupStatus : member.status;
 
-    const dialogRef = this.dialog.open(ChangeMemberStatusDialogComponent, config);
-    dialogRef.afterClosed().subscribe((changedMember: Member) => {
-      if (changedMember) {
-        const changedStatus: string = groupId ? changedMember.groupStatus : changedMember.status;
-        if (
-          (oldStatus === 'VALID' &&
-            (changedStatus === 'EXPIRED' || changedStatus === 'DISABLED')) ||
-          changedStatus === 'VALID'
-        ) {
-          if (groupId) {
-            member.groupStatus = changedStatus;
+    // Skip crating the status dialog, if the current group is 'members'
+    if (this.isMembersGroup && groupId) return;
+
+    this.memberManager.getRichMemberWithAttributes(member.id).subscribe((richMember) => {
```

Comment:
Don't request the attribute here but rather use the `attrNames` input of this component.